hpat/tests/test_dataframe.py

```diff
@@ -332,30 +333,32 @@ def test_impl(df, n):
         df2 = df.copy()
         pd.testing.assert_frame_equal(hpat_func(df, n), test_impl(df2, n))
 
-    @unittest.skip('AssertionError - fix needed\n'
-                   'Attribute "dtype" are different\n'
-                   '[left]:  int64\n'
-                   '[right]: int32\n')
     def test_set_column1(self):
-        # set existing column
-        def test_impl(n):
-            df = pd.DataFrame({'A': np.ones(n, np.int64), 'B': np.arange(n) + 3.0})
-            df['A'] = np.arange(n)
-            return df
-
+        if platform.system() == 'Windows' and not IS_32BITS:
+            # set existing column
+            def test_impl(n):
+                df = pd.DataFrame({'A': np.ones(n, np.int64), 'B': np.arange(n) + 3.0})
+                df['A'] = np.arange(n, np.int64)
+                return df
+        else:
+            # set existing column
+            def test_impl(n):
+                df = pd.DataFrame({'A': np.ones(n, np.int64), 'B': np.arange(n) + 3.0})
```

Comment:
I'm not sure, but is `np.int64` exactly needed here?

Reply:
@densmirn In which place? In dataframe creation it's not needed - yes, only in the assignment later. But it turns out that np.arange cannot be jitted with dtype arg anyway, so I'm re-writing this at the moment.